contracts/Registry.vy

```diff
@@ -99,7 +101,8 @@ def latestRelease() -> String[28]:
     @return The api version of the latest release.
     """
     # NOTE: Throws if there has not been a release yet
-    return Vault(self.releases[self.numReleases - 1]).apiVersion()  # dev: no release
```

Comment:
This is breaking the error reporting:
https://eth-brownie.readthedocs.io/en/stable/tests-pytest-intro.html?highlight=%23dev#developer-revert-comments